GitUI/CommitInfo/RefsFormatter.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using GitCommands;
+using ResourceManager;
+
+namespace GitUI.CommitInfo
+{
+    public sealed class RefsFormatter : ITranslate
+    {
+        /// <summary>
+        /// The number of displayed refs if the list is limited.
+        ///
+        /// If limited, the line "[Show all]" and an empty line are added.
+        /// Hence the list needs to be limited only if it exceeds MaximumDisplayedRefsIfLimited + 2.
+        /// </summary>
+        private const int MaximumDisplayedRefsIfLimited = 10;
+
+        private static readonly TranslationString _containedInBranches = new TranslationString("Contained in branches:");
+        private static readonly TranslationString _containedInNoBranch = new TranslationString("Contained in no branch");
+        private static readonly TranslationString _containedInTags = new TranslationString("Contained in tags:");
+        private static readonly TranslationString _containedInNoTag = new TranslationString("Contained in no tag");
```

Comment:
Can we not leverage the same mechanism for conditional strings as we have in `ResourceManager.Strings`?

Reply:
`RefsFormatter` would be a `static` class if it did not need to implement `ITranslate`.
Only temporary instance is created in order to satisfy the translation interface.

Is there a way in C# to give the metaobject to the `Translator`?
As an alternative, these few `TranslationString`s could be moved to `ResourceManager.Strings`.

Reply:
Is there a way in C# to give the metaobject to the `Translator`?

Should I move these few `TranslationString`s to `ResourceManager.Strings`, @RussKie?

Reply:
I have moved the `TranslationString`s to `Strings`.